src/components/addCredentialType/__tests__/__snapshots__/addCredentialType.test.js.snap

```diff
@@ -10,6 +10,10 @@ exports[`ToolbarFieldGranularity Component should export select options: fieldOp
     "title": [Function],
     "value": "openshift",
   },
+  {
+    "title": [Function],
+    "value": "acs",
```

Comment:
I don't know why (might be backend thing), but when I do Credentials -> Add Credential -> ACS credential, then "Source type" in new dialog window is "acs", all lowercase. I think it should be all uppercase, "ACS". All other types are correctly capitalized, like "Network" or "OpenShift".

![image](https://github.com/quipucords/quipucords-ui/assets/12373754/f6576140-bd90-4b8a-aae9-6bd2cea084a3)

Reply:
Around `line 60` in `en.json` if you add the key with context under `form-dialog`
```
"label_acs": "ACS",
```
... this will display the human intended display of "ACS" @nicolearagao . This should also apply to the Source and Credential displays.

The reason
You just happened to miss one string =). Because of the way "context" works on strings you're currently displaying the fallback context passed to the translate function... if you look at @bruno-fs old PR you'll catch the similar `Ansible` update, https://github.com/quipucords/quipucords-ui/pull/201/files#diff-1ef07a459de08da33c9f950f32e4e739683a016d219c16503bf8c3a5e2c973c7R173

Reply:
Appears to now be working as @mirekdlugosz  noted